internal/controllers/clusterextension_controller.go

```diff
@@ -257,10 +257,10 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 			setStatusUnpackFailed(ext, err.Error())
 			return ctrl.Result{}, err
 		}
-		setStatusUnpacked(ext, unpackResult.Message)
+		setStatusUnpacked(ext, fmt.Sprintf("unpack successful: %v", unpackResult.Message))
 	default:
-		setStatusUnpackFailed(ext, err.Error())
-		return ctrl.Result{}, err
+		setStatusUnpackFailed(ext, fmt.Sprintf("unpack successful: %v", unpackResult.Message))
```

Comment:
Seems like "unpack successful" is not meant to be here

Reply:
Ah! Good catch!

Reply:
I was wondering the same, but I believe it might negatively impact the tests... but we should evaluate.

Reply:
Yes. I was about to push the change. It should be:
```
	default:
		setStatusUnpackFailed(ext, "unexpected unpack status")
		// We previously exit with a failed status if error is not nil.
		return ctrl.Result{}, fmt.Errorf("unexpected unpack status: %v", unpackResult.Message)
```

Ran this fix locally and e2es work. Will push the change after a rebase.

Reply:
Addressed by https://github.com/operator-framework/operator-controller/pull/905/commits/6bc072fa5a44f2884c753b4809c0bb16aceb1f6c